test/FastTests/Server/Documents/Replication/UtilityTests.cs

```diff
@@ -21,21 +34,24 @@ public class UtilityTests : RavenTestBase
                 DocumentsOperationContext context;
                 using (db.DocumentsStorage.ContextPool.AllocateOperationContext(out context))
                 {
+                    var guid1 = Guid.NewGuid();
+                    var guid2 = Guid.NewGuid();
+                    var guid3 = Guid.NewGuid();
                     var collection = new[]
                     {
                         new DynamicJsonValue
                         {
-                            ["Key"] = new DynamicJsonArray(Guid.NewGuid().ToByteArray().Cast<object>()),
+                            ["Key"] = new DynamicJsonArray(guid1.ToByteArray().Cast<object>()),
```

Comment:
Why is this stored as an array? 
It would be better to have it as a raw string.